tests/test_send_sms.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+from app import db, create_app
+
+from app.controllers.sms_send import send_sms
+
+PHONE_NUMBER = "+380634136800"
+
+
+@pytest.fixture
+def client():
+    app = create_app(environment="testing")
+    app.config["TESTING"] = True
+
+    with app.test_client() as client:
+        app_ctx = app.app_context()
+        app_ctx.push()
+        db.drop_all()
+        db.create_all()
+        yield client
+        db.session.remove()
+        db.drop_all()
+
+
+def test_send_sms(client):
```

Comment:
this test can be run only if set correct PHONE_NUMBER!
If PHONE_NUMBER is None - this test must be ignored...
Add decorator:
```python
@pytest.mark.skipif(PHONE_NUMBER is None, reason="Need set test phone number")

Reply:
OK